package.json

```diff
@@ -3,7 +3,7 @@
     "displayName": "Markdown All in One",
     "description": "All you need to write Markdown (keyboard shortcuts, table of contents, auto preview and more)",
     "icon": "images/Markdown-mark.png",
-    "version": "1.2.0",
+    "version": "1.2.1",
```

Comment:
We don't need to manually change the version number because `vsce` will handle this when publishing extension.